
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>COVID19-RAGE</title>
    <link rel="stylesheet" href="./style.css">
    <link rel="icon" href="./favicon.ico" type="image/x-icon">
  </head>
  <body>
    <main>
        <h1>scRNA-seq analysis of PBMC from COVID-19 patients of the Padova cohort</h1>
        <p>The code and the data in this repository enables the reproduction of the analyses and plots for 
the single-cell RNA-seq data in the paper "RAGE engagement by SARS-CoV-2 enables monocyte infection and
underlies COVID-19 severity" (https://doi.org/10.1101/2022.05.22.492693).</p>
    <p>

<h2><a href=01_gex_filtering_annotation.html>Filtering, dimensionality reduction, cell annotation</a></h2><p>Notebook that containes the preliminary steps of the single-cell data analysis</p>
<p>1) initial quality filters 
2) log-normalization of the counts and HVG selection
3) dimensionality reduction (PCA and UMAP) with Harmony integration 
4) Leiden clustering and celltype annotation
5) preliminary analysis and visualization of the annotated data</p>
<h2><a href=02_abundance_analysis_cell_families.html>Differential abundance analysis</a></h2><p>RMarkdown code for the computation of the differential abundance of the
cell families that have been defined in the dataset.</p>
<h2><a href=03_dataset_exploration_and_ingestion.html>Data exploration and ingestion of external annotations</a></h2><p>Plots of the relevant cell metadata and
comparison with ingested cell annotations based on
<a href="https://www.nature.com/articles/s41591-020-0944-y">Wilk et al.</a></p>
<h2><a href=04_pseudobulk.html>Pseudobulk data computation</a></h2><p>Computation of the pseudobulk data for each patient sample</p>
<h2><a href=05_edgeR_DE.html>Differential expression analysis</a></h2><p>Differential expression analysis of the pseudobulk data
with edgeR GL model</p>
<h2><a href=06_RAGE_pathway_exploration.html>RAGE pathway exploration</a></h2><p>Supervised exploration of the expression of the 
genes included in the RAGE binding gene list</p>
<h2><a href=07_external_dataset_summary.html>External dataset summary</a></h2><p>Summary of the results of the RAGE exploration
performed on the publicly available datasets of 
single-cell RNA-seq of COVID-19 patients</p>
<p>Detailed analisys workflows are described in the following notebooks:</p>
<ul>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_arunachalam_2020_processed.html">Arunachalam et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_silvin_2020_processed.html">Silvin et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_yao_2021_processed.html">Yao et al. 2021</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_bost_2021_processed.html">Bost et al. 2021</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_stephenson_2021_processed.html">Stephenson et al. 2021</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_yu_2020_processed.html">Yu et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_combes_2021_processed.html">Combes et al. 2021</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_su_2020_processed.html">Su et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_zhu_2020_processed.html">Zhu et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_lee_2020_processed.html">Lee et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_wen_2020_processed.html">Wen et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_schulte-schrepping_2020_processed.html">Schulte-Schrepping et al. 2020</a></li>
<li><a href="https://giuseppetestalab.github.io/covidiamo-padova/08_wilk_2020_processed.html">Wilk et al. 2020</a></li>
</ul>

      <h2>Drug repurposing analysis</h2>
<p>Links: <a href="https://github.com/GiuseppeTestaLab/covidiamo-padova/tree/main/10_drug_repurposing_analysis">folder</a>.</p>
<p>Analysis of the transcriptomic signatures based on the Connectivity Map database.
This resource has been used to:</p>
<ul>
<li>detect whether current COVID-19 therapeutics are were able to mimic the transcriptional
  changes occurring from in the subjects of the study from admission to recovery;</li>
<li>determine which drugs included in the CMap database are able to interfere 
  specifically with the RAGE pathway;</li>
<li>carry our an unbiased analysis of the CMap database, seeking compounds potentially 
  capable of reverting severity signatures across cell families.</li>
</ul>
    </p>
    </main>
    <script src="index.js"></script>
  </body>
</html>

